### ISSUE TITLE
Resource form misses license, language, format and more

### ISSUE THREAD

**Issue by wenzeslaus** [OSGeo/osgeo#18]
In comparison to the edu content inventory (e.g. http://www.osgeo.org/node/1592), the resource form does not have a field for a:

* [ ] license (e.g. can I reuse commercially?)
* [ ] (natural) language
* [ ] format (that's good to know what you will get when you get there; a web page, website, GitHub repo with Markdown files, PDF, ...?).

The level of experience (Graduate Students, Intermediate GIS Users, Mid-career, ...) and multiple URLs are also missing, but I can see good reasons to leave them out (e.g. unclear definitions).

**Comment by jodygarnett** [comment]
Those are some good suggestions, I also want to make sure that it is easy/quick to add content so we should balance the number of fields we require.

Would it be okay to have a "spec" section like projects where you can list details such as you describe? Or does it need to be structured.

**Comment by wenzeslaus** [comment]
I don't think it needs to be structured much, but it can be on some Spec page. Pages actually helped me with filling.

It definitively should be optional. However, searching by a license is an important case and we should be care about licensing of these in the same way we care about licenses for code. Having them as explicit (rather than fields) I think may encourage people to actually fill it if they know the answer. Some radio buttons e.g. "CC BY, CC BY-SA, ... + Not specified + Other:" for license would make filling faster. Minimizing the number of fields makes sense, but I would prefer to see Language rather than Discipline which seem to me quite restrictive.

I'm actually fine with more and overlapping fields because it may improve search or systematic sorting, but if we are trying to minimize them, I think there is some overlap. Community, Tags and Initiative have similar descriptions (I think I ended up putting GeoForAll everywhere). Obviously, anything/everything can go to Tags, so I actually don't know what to do with this field. Again, if it is from GeoForAll community seems to be of lesser importance for the user than if they can take it, modify it, and resuse it.

**Comment by jodygarnett** [comment]
Confirmed, editing - http://osgeo.getinteractive.nl/resources/geo-spatial-org/

**Comment by jodygarnett** [comment]
For multiple URLs you should be able to add multiple call to actions, I have done so in a few places.

Example: http://osgeo.getinteractive.nl/resources/osgeo-agm-2017/
